src/openeo_test_suite/lib/compliance_util.py

```diff
@@ -336,19 +345,18 @@ def put_process_graphs(base_url: str, bearer_token: str):  # TODO id and so fort
 
     try:
         for payload in payloads:
-            id, payload = set_uuid_in_udp(payload)
+            id = str(uuid.uuid4().hex)
             created_udp_ids.append(id)
-            response = requests.put(
-                f"{base_url}process_graphs/{id}",
+            requests.put(
+                f"{base_url}/process_graphs/{id}",
                 data=payload,
                 headers={
                     "Content-Type": "application/json",
                     "Authorization": f"{bearer_token}",
                 },
             )
-            print(response)
     except Exception as e:
-        _log.error(f"Failed to create process graph: {e}")
+        print(f"Failed to create process graph: {e}")
```

Comment:
just doing a print on any exception seems like asking for trouble down the line. Why not fail hard just here?